### DISCUSSION TITLE
Oracle with Hammerdb, test results TPM is 0

### DISCUSSION THREAD

**Discussion by springeee** [TPC-Council/HammerDB#155] [Q&A] [answered]
Hi all:
    I use Hammerdb test Oracle , my step as follows:
1, install Oracle 19c on Linux
2, create tablespace tpcctab and user
SQL> create bigfile tablespace tpcctab datafile '/mnt/dp32t/tpcctab.dbf' size 180g;

SQL> create user tpcc identified by tpcc default tablespace tpcctab;
3, open Hammerdb,  Schema Build
![image](https://user-images.githubusercontent.com/72604063/95569503-3f47c500-0a58-11eb-9d36-0b989103518b.png)

4, Driver Options
![image](https://user-images.githubusercontent.com/72604063/95569629-67cfbf00-0a58-11eb-9b48-4ad10c975003.png)

5, Virtual User Options
![image](https://user-images.githubusercontent.com/72604063/95569693-7ae28f00-0a58-11eb-98e3-5044a7f4e518.png)

6, run
![image](https://user-images.githubusercontent.com/72604063/95570019-ee849c00-0a58-11eb-970a-6682713a226e.png)

7, output
![image](https://user-images.githubusercontent.com/72604063/95570132-0eb45b00-0a59-11eb-95bc-88b0c16c55c5.png)
from the output we can see the Oracle TPM is 0.
I don't know why.

Please help to give me some advices?
Thanks.

**Comment by sm-shaw** [comment, marked as answer]
Hi,
From the instance name it looks like you are using a testing a pluggable database? If so this is a known issue recorded in this issue here:  "Oracle 12.2 multitenant" #13. Oracle changed some of the view names in 12.2 that broke backward compatibility with some of the queries that previously worked against pluggable databases TPM being one of them.  It should be possible to provide the system connection to the CDB and get the same statistics that were available previously.  NOPM is extracted from the schema so is not affected by the view name changes. 
Steve

**Comment by springeee** [comment]
Hi Steve:

Thanks for your reply.
now use CDB , the TPM result is ok.

Thanks.
